<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import Data</class>
 <widget class="QMainWindow" name="Import Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>범주화</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="arrBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>851</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="title">
     <string>범위 지정</string>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>44</height>
      </rect>
     </property>
     <property name="text">
      <string>최소 범위</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="minText">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>91</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>101</width>
       <height>44</height>
      </rect>
     </property>
     <property name="text">
      <string>최대 범위</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>101</width>
       <height>44</height>
      </rect>
     </property>
     <property name="text">
      <string>범주 간격</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="maxText">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>91</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="interText">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>91</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>709</x>
       <y>30</y>
       <width>120</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>실행</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>861</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>570</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt; 이전</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>완료</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>861</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>비식별 처리 기법 : 범주화</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="arrBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>851</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="title">
     <string>원본 데이터 분포</string>
    </property>
    <widget class="MplWidget" name="i_catGraph" native="true">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>12</y>
       <width>851</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[ : &lt;= ,   ) : &gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
